core/storage/couchStorage_test.go

```diff
@@ -0,0 +1,95 @@
+package storage
+
+import (
+	"github.com/open-horizon/edge-sync-service/common"
+	"testing"
+	"time"
+)
+
+func TestCouchStorageConnection(t *testing.T) {
+	store, err := setUpStorage(common.Couch)
+	if err != nil {
+		t.Errorf(err.Error())
+		return
+	}
+	defer store.Stop()
+}
+
+func TestStorageObjects(t *testing.T) {
+	store, err := setUpStorage(common.Couch)
+	if err != nil {
+		t.Errorf(err.Error())
+		return
+	}
+	defer store.Stop()
+
+	tests := []struct {
```

Comment:
I didn't call testStorageObjects() because that test calls some other functions which I have not yet implemented. I also wanted to create my own structure because I wanted to test all the possible scenarios with/without binary data, different values of NoData and MetaOnly flag, etc. specific to the functions I had implemented. As I implement more functions, I will make unit tests consistent with how they have been implemented for mongo, i.e calling functions from storage_test.go. Would that be okay?